app/controllers/application_controller.rb

```diff
@@ -1,5 +1,14 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
-  protect_from_forgery with: :exception
+  protect_from_forgery unless: -> { request.format.json? }
```

Comment:
this is ok, but ideally we would create an `ApiController` that all of our Api controllers will instead inherit from!
```ruby
# api_controller.rb

class ApiController < ActionController::Base
  protect_from_forgery unless: -> { request.format.json? }
end
```